requirements.txt

```diff
@@ -2,10 +2,11 @@
 colour-runner==0.0.4
 coverage==4.2
 dj-database-url==0.4.1
-django>=1.8,<1.9
+django>=1.10,<1.11
```

Comment:
Given `requirements.txt` is only used for tests, and the version of django is determined by the text matrix, might it be best to remove django from `requirements.txt` altogether?